README.rst

```diff
@@ -34,8 +35,72 @@ To install from source:
 
 .. installation-end-marker
 
-Next Steps
-----------
+Getting Started
+===============
+
+Sign up for the Leap quantum cloud service here:
+`Leap signup <https://cloud.dwavesys.com/leap/signup>`_. Leap provides a
+browser-based integrated development environment (IDE) with Ocean preinstalled.
+
+Start learning with the following D-Wave resources:
+
+* `System Documentation <https://docs.dwavesys.com/docs/latest/index.html>`_ to
+  learn about quantum computers and how to use them.
+
+* `Getting Started with Ocean <https://docs.ocean.dwavesys.com/en/stable/getting_started>`_
+  to install and start coding with Ocean software.
+
+* `dwave-examples <https://github.com/dwave-examples>`_ for code examples
+  and Jupyter Notebooks.
+
+* `D-Wave website <https://www.dwavesys.com/learn/resource-library>`_ for
+  whitepapers and additional resources.
+
+Example Quantum Program
+-----------------------
+
+The following lines of code solve and visualize a
+`random <https://docs.ocean.dwavesys.com/en/stable/docs_dimod/reference/generators.html>`_
+problem on a quantum computer.
+
+>>> from dwave.system import DWaveSampler, EmbeddingComposite
+>>> from dimod.generators import ran_r
```

Comment:
I wonder if there is a more explicitly named problem generator we can use. Maybe `maximum_independent_set()`? Or even `gnm_random_bqm()`? The latter is at least following NumPy's naming convention.

I find `ran_r` a bit inscrutable.

Reply:
This is the best replacement I've found so far, `gnp_random_bqm(25, .25, 'BINARY')`, and I find the current graphic better looking. The networkx-based problems need a bit more setup, which I'd prefer to avoid. Note the link is to the dimod generators page, so if curious users look up `ran_r`, there's a benefit of acquaintance with our generators.  

<img width="566" alt="image" src="https://user-images.githubusercontent.com/34041130/144515692-60a03c93-a343-4fbd-98da-7916863ed8a4.png">

Reply:
Fair enough, thanks for trying